src/Pages/Cart/Cart.css

```diff
@@ -0,0 +1,70 @@
+.cart-component {
+    display: flex;
+    justify-content: center;
+    padding: 0.5rem;
+    flex-wrap: wrap;
+  }
+  
+  .cart-component .product-detail {
+    margin: auto;
+  }
+  
+  .cart-component .product-name{
+    font-weight: bold;
+  }
+  
+  .cart-component .footer {
+    display: flex;
+    flex-direction: column;
+    margin-top: 0rem !important;
```

Comment:
If there is any alternative way to override this, please prefer that rather then !important